_topic_maps/_topic_map_ms.yml

```diff
@@ -53,8 +53,8 @@ Distros: microshift
 Topics:
 - Name: Installing from RPM
   File: microshift-install-rpm
-#- Name: Installing in a RHEL for Edge Image
-#  File: microshift-install-rhel-for-edge
+- Name: Embedding into a RHEL for Edge image
```

Comment:
Not a native-speaker, just wanted to add that when I proposed "embed into" it was because the M-W examples are about the end-state of being embedded in, while the examples I've found for describing the "movement from one state to the state of being embedded" used "embed into".

Reply:
Noted. I should have clarifited that this is just a small nit. Definitely not a blocker.

Reply:
More dictionaries and styles (including Oxford) do side with "in", so I changed it. Although I am with Frank on preference for the nuance of "into."